### PR TITLE
Show item charges while on ground

### PR DESCRIPTION
**Pull request by Marsunpaisti**
Appends "(x itm.getCharges())" on items when dropped, and removes the suffix on pickup

### CODE REVIEW

File: wurst/systems/core/ItemAmount.wurst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package ItemAmount
+
+import ClosureEvents
+import StackNSplit
+
+function onDrop(item itm)
+    if isItemStackable(itm.getTypeId())
+        var name = itm.getName()
+        int foundIndex = name.lastIndexOf(" (x")
```
Comment (jonathanmclaus) [inline wurst/systems/core/ItemAmount.wurst]: Abstract lines 9 - 11 into a function. You're doing the exact same thing in the `onPickup`.

Comment (Marsunpaisti) [reply wurst/systems/core/ItemAmount.wurst]: updated

File: wurst/systems/core/ItemAmount.wurst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+function getItemBaseName(item itm) returns string
+    let name = itm.getName()
+    int foundIndex = name.lastIndexOf(" (x")
+    if foundIndex != -1
+        return name.substring(0, foundIndex)
+
+    return name
+
+init
+    registerPlayerUnitEvent(EVENT_PLAYER_UNIT_PICKUP_ITEM) ->
```
Comment (jonathanmclaus) [inline wurst/systems/core/ItemAmount.wurst]: Are these two events enough to cover all cases, like unit death and quickdrop?

Comment (Marsunpaisti) [reply wurst/systems/core/ItemAmount.wurst]: Unit.RemoveItem causes drop events so yes

**Pull request merged**